ceylon-mode.el

```diff
@@ -95,35 +95,42 @@
     st)
   "Syntax table for `ceylon-mode'.")
 
-(set-default 'tab-width 4)
+(set-default 'tab-width                     4)
+(set-default 'ceylon-return-point-on-indent nil)
```

Comment:
It's funny; I usually set these things in my `.emacs` so I hadn't even thought of that. I agree giving the option for otherwise is the better choice, though.